anmonzi/barkbook
# Dog cards populate when location is clicked

Dog cards populate when a single location is clicked. Image, energyLevel, and size render, and user can click on dog card to expand a more detailed profile (which is in the works).

## Steps to Test
1. git fetch --all
2. git checkout populateLocation-withUsers
3. reload page
4. click on a location and verify dog cards render with photo, name, energy level, and size. 
5. Verify you can click on a dog card and be taken to another profile page.
